### ISSUE TITLE
java.lang.ClassCastException Long

### ISSUE THREAD

**Issue by passivedragon** [North-West-Wind/CurseForge-CLI#3]
I'm on Linux Mint, after downloading the latest available release (1.2.2), I extracted the .zip into a folder and ran
`./curseforge modpack install 454031`

This returns:
```log
$ ./curseforge modpack install 454031
CurseForge CLI v1.2.2
New version available: 1.2.1
Added acceptParent option
acceptParent allows user to install mods supporting only the parent version of a profile. Refer to the README in the repository for more details.
Installing Crucial 2 - The Refresh Update...
java.lang.ClassCastException: class java.lang.Long cannot be cast to class java.lang.Integer (java.lang.Long and java.lang.Integer are in module java.base of loader 'bootstrap')
	at ml.northwestwind.Modpack.lambda$getManifestFile$2(Modpack.java:120)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:176)
	at java.base/java.util.ArrayList$ArrayListSpliterator.tryAdvance(ArrayList.java:1632)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:127)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:502)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:488)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:543)
	at ml.northwestwind.Modpack.getManifestFile(Modpack.java:120)
	at ml.northwestwind.Modpack.install(Modpack.java:59)
	at ml.northwestwind.Modpack.run(Modpack.java:31)
	at ml.northwestwind.Main.main(Main.java:28)
```

Do I need to ensure a specific Java version?

**Comment by WampiFlampi** [comment]
Same issue here, would appreciate build instructions for this project I'd be happy to troubleshoot this myself

**Comment by North-West-Wind** [comment]
Sorry for the wait. Will be fixed soon

**Comment by North-West-Wind** [comment]
The issue is fixed in v1.2.3, but errors may still pop up when installing modpacks you've tried to install with v1.2.2.
To solve the problem, you can go into `{directory}/modpack` (substitute `{directory}` with the working directory of CurseForge CLI, which can be found inside `cf.json`) and delete the folder of modpack you tried to install.
